### ISSUE TITLE
Would it be possible to update IntelliJ installation steps with StackOverflow ones?

### ISSUE THREAD

**Issue by dzuluaga** [google/google-java-format#351]
Hello, I found rather hard to follow these steps. Maybe they apply to older versions of IntelliJ. But the latest version of IntelliJ has a way to install plugins directly from IntelliJ.

> The plugin will be disabled by default. To enable it in the current project, go to File→Settings...→google-java-format Settings (or IntelliJ IDEA→Preferences...→Other Settings→google-java-format Settings on macOS) and check the Enable google-java-format checkbox. (A notification will be presented when you first open a project offering to do this for you.)

These steps are better described in this [StackOverflow answer](https://stackoverflow.com/questions/42979700/how-to-configure-google-java-code-formatter-in-intellij-idea-17).

Thanks!

**Comment by plumpy** [comment]
Updated the instructions to mention how to download it from the plugin marketplace. Thanks for the request.
